tests/src/test/scala/com/nvidia/spark/rapids/GpuShuffledHashJoinExecSuite.scala

```diff
@@ -58,6 +60,7 @@ class GpuShuffledHashJoinExecSuite extends AnyFunSuite with MockitoSugar {
     when(mockStreamIter.hasNext).thenReturn(true)
     val (builtData, _) = GpuShuffledHashJoinExec.prepareBuildBatchesForJoin(
       buildIter,
+      GpuColumnVector.dataTypesFromAttributes(buildAttrs.asJava),
```

Comment:
Here's an example where the Java API makes this harder than it should be.  This doesn't require an import or a Java conversion:
```suggestion
     buildAttrs.map(_.dataType).toArray,
```